docs/data/date-pickers/adapters-locale/CustomDayOfWeekFormat.tsx

```diff
@@ -12,7 +12,7 @@ export default function CustomDayOfWeekFormat() {
       <DateCalendar
         value={value}
         onChange={(newValue) => setValue(newValue)}
-        dayOfWeekFormatter={(day) => `${day}.`}
+        dayOfWeekFormatter={(_day, weekday) => `${dayjs(weekday).format('dd')}.`}
```

Comment:
For me `weekday` is already a `dayjs` object so we don't need to re-parse it.
Do you have issues when removing it?

```suggestion
        dayOfWeekFormatter={(_day, weekday) => `${weekday.format('dd')}.`}
```